CrossrefReferenceLinkingInfoSender.inc.php

```diff
@@ -74,15 +74,19 @@ function _getJournals() {
 		PluginRegistry::loadCategory('importexport');
 		$crossrefExportPlugin = PluginRegistry::getPlugin('importexport', 'CrossRefExportPlugin');
 
-		$contextDao = Application::getContextDAO(); /* @var $contextDao JournalDAO */
+		$contextDao = Application::getContextDAO(); /** @var $contextDao JournalDAO */
 		$journalFactory = $contextDao->getAll(true);
 		$journals = array();
 		while($journal = $journalFactory->next()) {
 			$journalId = $journal->getId();
-			if (!$journal->getSetting('citationsEnabledSubmission') || !$crossrefExportPlugin->getSetting($journalId, 'username') || !$crossrefExportPlugin->getSetting($journalId, 'password') || !$crossrefExportPlugin->getSetting($journalId, 'automaticRegistration')) continue;
-			$journals[] = $journal;
+			if ($this->_plugin->citationsEnabled($journalId) &&
+				$crossrefExportPlugin->getSetting($journalId, 'username') &&
+				$crossrefExportPlugin->getSetting($journalId, 'password') &&
+				$crossrefExportPlugin->getSetting($journalId, 'automaticRegistration')) {
```

Comment:
we should then remove this condition so that the scheduled task is always executed